broker/value_added/classify.py

```diff
@@ -0,0 +1,91 @@
+from astrorapid.classify import Classify
+
+
+def rapid(light_curves, cx_dicts, plot=False, plotdir='./broker/value_added/plots',
```

Comment:
You can remove the `plot` argument here and just use the `plot_dir` argument. I don't think it's appropriate for `plot_dir` to have a default value here.